bin/hysr_visualization

```diff
@@ -76,7 +76,8 @@ def _get_handle_to(mujoco_id,
     table = pam_mujoco.MujocoTable(segment_id_table,
                                    position=table_position,
                                    orientation=table_orientation)
-    robot = pam_mujoco.MujocoRobot(segment_id_robot,
+    robot = pam_mujoco.MujocoRobot(False,
```

Comment:
Since the `False` alone is not very descriptive, how about using named arguments?  I.e. like this:

    pamy1=False,
    segment_id=segment_id_robot,
    ...